doctest/docs/ppl/curl.rst

```diff
@@ -0,0 +1,41 @@
+Example
+-------
+
+Test query::
```

Comment:
I assume the 3 docs added in this PR (`ppl/cli/rst`, `ppl/curl.rst`, `sql/basics.rst`) are more like samples? Never mind if we plan to change it to real docs exposed to customer. If not, probably we can move them to a `sample` folder used to show how to add doctest by CLI or curl for PPL/SQL.

Reply:
Yeah my plan is to move it to real docs in next PR with clear documentation and examples of PPL. Still need to figure out the layout of docs. Should we add subfolder of `ppl`, or just add a single file for now. 
Mixing sql and ppl code examples together in one doc doesn't work with doctest now, because we setup different client with different query language.

Reply:
Moved to `sample` folder under `docs`

Reply:
Thanks!